src/ring_theory/etale.lean

```diff
@@ -81,13 +81,14 @@ formally_etale.iff_unramified_and_smooth.mpr ⟨h₁, h₂⟩
 
 omit R A
 
-lemma formally_unramified.lift_unique [formally_unramified R A] (I : ideal B)
+lemma formally_unramified.lift_unique {B : Type u} [comm_ring B] [_RB : algebra R B]
+  [formally_unramified R A] (I : ideal B)
   (hI : is_nilpotent I) (g₁ g₂ : A →ₐ[R] B) (h : (ideal.quotient.mkₐ R I).comp g₁ =
   (ideal.quotient.mkₐ R I).comp g₂) : g₁ = g₂ :=
 begin
   revert g₁ g₂,
   change function.injective (ideal.quotient.mkₐ R I).comp,
-  unfreezingI { revert _inst_5 },
+  unfreezingI { revert _RB },
```

Comment:
I thought I commented this already, but I can't find the comment. Does this work? 
```suggestion
  unfreezingI { revert ‹_› },
```

Reply:
Nope, that was my initial attempt as well.